components/dependency_injection/autowiring.rst

```diff
@@ -0,0 +1,411 @@
+.. index::
+    single: DependencyInjection; Autowiring
+
+Using the Autowiring
+====================
+
+Autowiring allows to register services in the container with minimal configuration.
+It is practical in the field of Rapid Application Development, when designing prototypes
+and in early stages of large projects. It makes it easy to bootstrap an app service
+graph and eases refactoring.
+
+Let’s see how it works. To do so we will build a fake API publishing statutes on a Twitter
```

Comment:
and you should avoid the first person in the Symfony docs (except from Quick Tour & Best practices guides). So you can write these 2 sentences like:

```rst
Imagine you're building an API to publish statuses on a Twitter feed, obfuscated with ROT13.
```